### DISCUSSION TITLE
Foooucs crashing causing computer to restart

### DISCUSSION THREAD

**Discussion by robinwang25** [lllyasviel/Fooocus#3212] [Q&A] [answered]
Since upgrading Foooucs in April, my computer frequently crashes and restarts while rendering images. There is no error message, and now I'm afraid to even use Foooucs because it always crashes. Before the upgrade, even if it was slow, it never crashed. I'm not sure what the problem is, and there are no signs of high GPU memory usage. Could you please advise on what might be causing this issue?

![1719746036189](https://github.com/lllyasviel/Fooocus/assets/141383987/469868bf-63d3-4743-9a79-788d1e0eea6b)

![1719747441796](https://github.com/lllyasviel/Fooocus/assets/141383987/b45de03a-8597-40d9-9e07-976d3d386a7c)

--------------------------------------------------------------------------
E:\Fooocus中文版国内在线更新>.\python_embeded\python.exe -s Fooocus\entry_with_update.py --always-download-new-model
已经是最新版本
国内在线更新成功！Fooocus中文本地化由AIHUB(aihub.videosreel.com)进行技术支持.
Fooocus2.4.2更新内容：
1.添加对 playground v2.5 的支持和预设（仅适用于性能质量或速度，与调度程序edm_playground_v2一起使用）
2.playground v2.5已设置为通过选择后自动进行下载
3.使文本框（提示词栏）可调整大小
4.文件刷新时使用默认的VAE名称而不是None
5.修复了历史日志的LoRA显示
6.修复了一些小错误（gamma 为 0 时的 tcd 调度程序、Dockerfile 中的 chown、自述文件中的 cmd args、纵横比的转换、文件重新加载后的 vae 默认值）

小贴士：
1. 以文本文档编辑bat启动文件，在第一行后面空格加--share，保存后运行即可生成公链分享给其他人使用Fooocus。
2. 控制台默认显示局域网程序运行地址，可以复制到其他设备进行使用出图。
3. 社区以去中心化进行模型和其他资源分享，去中心化下载完全免费并且不受限速。
4. 社区更新了Omost。

[System ARGV] ['Fooocus\\entry_with_update.py', '--always-download-new-model']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.4.3
Failed to load config key: {"path_checkpoints": "D:\\BaiduNetdiskDownload\\Fooocus_win64_2.1.789\u56fd\u5185\u5728\u7ebf\u66f4\u65b0\\Fooocus\\models\\checkpoints"} is invalid or does not exist; will use {"path_checkpoints": ["../models/checkpoints/"]} instead.
Failed to load config key: {"path_loras": "D:\\BaiduNetdiskDownload\\Fooocus_win64_2.1.789\u56fd\u5185\u5728\u7ebf\u66f4\u65b0\\Fooocus\\models\\loras"} is invalid or does not exist; will use {"path_loras": ["../models/loras/"]} instead.
Failed to load config key: {"path_embeddings": "D:\\BaiduNetdiskDownload\\Fooocus_win64_2.1.789\u56fd\u5185\u5728\u7ebf\u66f4\u65b0\\Fooocus\\models\\embeddings"} is invalid or does not exist; will use {"path_embeddings": "../models/embeddings/"} instead.
Failed to load config key: {"path_vae_approx": "D:\\BaiduNetdiskDownload\\Fooocus_win64_2.1.789\u56fd\u5185\u5728\u7ebf\u66f4\u65b0\\Fooocus\\models\\vae_approx"} is invalid or does not exist; will use {"path_vae_approx": "../models/vae_approx/"} instead.
Failed to load config key: {"path_upscale_models": "D:\\BaiduNetdiskDownload\\Fooocus_win64_2.1.789\u56fd\u5185\u5728\u7ebf\u66f4\u65b0\\Fooocus\\models\\upscale_models"} is invalid or does not exist; will use {"path_upscale_models": "../models/upscale_models/"} instead.
Failed to load config key: {"path_inpaint": "D:\\BaiduNetdiskDownload\\Fooocus_win64_2.1.789\u56fd\u5185\u5728\u7ebf\u66f4\u65b0\\Fooocus\\models\\inpaint"} is invalid or does not exist; will use {"path_inpaint": "../models/inpaint/"} instead.
Failed to load config key: {"path_controlnet": "D:\\BaiduNetdiskDownload\\Fooocus_win64_2.1.789\u56fd\u5185\u5728\u7ebf\u66f4\u65b0\\Fooocus\\models\\controlnet"} is invalid or does not exist; will use {"path_controlnet": "../models/controlnet/"} instead.
Failed to load config key: {"path_clip_vision": "D:\\BaiduNetdiskDownload\\Fooocus_win64_2.1.789\u56fd\u5185\u5728\u7ebf\u66f4\u65b0\\Fooocus\\models\\clip_vision"} is invalid or does not exist; will use {"path_clip_vision": "../models/clip_vision/"} instead.
Failed to load config key: {"path_fooocus_expansion": "D:\\BaiduNetdiskDownload\\Fooocus_win64_2.1.789\u56fd\u5185\u5728\u7ebf\u66f4\u65b0\\Fooocus\\models\\prompt_expansion\\fooocus_expansion"} is invalid or does not exist; will use {"path_fooocus_expansion": "../models/prompt_expansion/fooocus_expansion"} instead.
[Cleanup] Attempting to delete content of temp dir C:\Users\Administrator\AppData\Local\Temp\fooocus
[Cleanup] Cleanup successful
Total VRAM 5120 MB, total RAM 65369 MB
Set vram state to: NORMAL_VRAM
Always offload VRAM
Device: cuda:0 Quadro P2200 : native
VAE dtype: torch.float32
Using pytorch cross attention
Refiner unloaded.
IMPORTANT: You are using gradio version 3.41.2, however version 4.29.0 is available, please upgrade.
--------
Running on local URL:  http://192.168.0.107:7865

To create a public link, set `share=True` in `launch()`.
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.logit_scale'}
Base model loaded: E:\Fooocus中文版国内在线更新\Fooocus\models\checkpoints\juggernautXL_v8Rundiffusion.safetensors
VAE loaded: None
Request to load LoRAs [('sd_xl_offset_example-lora_1.0.safetensors', 0.1)] for model [E:\Fooocus中文版国内在线更新\Fooocus\models\checkpoints\juggernautXL_v8Rundiffusion.safetensors].
Loaded LoRA [E:\Fooocus中文版国内在线更新\Fooocus\models\loras\sd_xl_offset_example-lora_1.0.safetensors] for UNet [E:\Fooocus中文版国内在线更新\Fooocus\models\checkpoints\juggernautXL_v8Rundiffusion.safetensors] with 788 keys at weight 0.1.
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cpu, use_fp16 = False.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
Started worker with PID 10580
程序成功启动. 使用程序通过 http://192.168.0.107:7865/ 或 192.168.0.107:7865

**Comment by mashb1t** [comment, marked as answer]
Please ensure your swap is on the same drive as Fooocus, at least this is what caused similar issues for other users: https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md#system-swap

**Reply by robinwang25** [reply]
![1719749477873](https://github.com/lllyasviel/Fooocus/assets/141383987/a9b55ab0-5481-4876-aed5-fb5cada2399c)
It's like this

**Reply by mashb1t** [reply]
try to disable automatic allocation and manually select the same drive as Fooocus is on.

**Comment by CDJenson** [comment]
I've got the same issue(2060,6GB VRAM,16GB RAM).I've spotted a pattern: "Loading 1 new model" causes my memory to spike to 99%, and then it switches to "loading in low VRAM mode xxx." ,sometimes, this even leads to my computer restarting unexpectedly.I have tried to downgrade the driver to versions 551.86 and 546.33, but it didn't work. I also tried to downgrade Fooocus to "previous_old_xformers_env.7z", and it didn't work either.I remember a similar problem occurred before and was solved, but I don't remember exactly which version it was.Finally, I want to thank all the contributors of the fooocus project for their efforts.

**Reply by mashb1t** [reply]
Please ensure you have swap set up correctly so RAM can outsource to it.

**Reply by CDJenson** [reply]
> Please ensure you have swap set up correctly so RAM can outsource to it.

Yeah, I can make sure of that. 
![image](https://github.com/lllyasviel/Fooocus/assets/22700161/2b6c5c14-dd07-4361-82d8-a62a734644c9)

I noticed another phenomenon today that occurs in LCM, Lightning, and Hyper-SD modes :when the memory reaches 99%, the fooocus program sometimes gets stuck,and the cmd window stops showing new logs,and then if i hit Enter,it resumes execution and the memory returns to normal.
ps: the LCM mode caused my computer to reboot twice today.

**Reply by mashb1t** [reply]
@robinwang25 Seems like your PC can't fully handle the load, I'm sorry.
